lib/core/src/server/config.js

```diff
@@ -58,6 +59,87 @@ function informAboutCustomConfig(defaultConfigName) {
   logger.info(`=> Using default webpack setup based on "${defaultConfigName}".`);
 }
 
+// Copied and modified from
```

Comment:
Please extract all these to a separate file.

Reply:
OK, I will extract them into `webpack_config`. Is it what you expect?